INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/authentication/login
ERROR    SWAGGER_COVERAGE_TRACKER:core.py:49 Unable to build endpoint coverage for HTTPX: 'ExercisesClient' object has no attribute 'request'
INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/files
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/files
INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/courses
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/courses
INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/exercises
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/exercises
INFO     HTTP_CLIENT:event_hooks.py:28 Make DELETE request to http://localhost:8000/api/v1/exercises/77c9b3c4-ab39-4a69-9da9-77c112548113
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/exercises/77c9b3c4-ab39-4a69-9da9-77c112548113
INFO     BASE_ASSERTIONS:base.py:19 Check that response status code equals to 200
INFO     HTTP_CLIENT:event_hooks.py:28 Make GET request to http://localhost:8000/api/v1/exercises/77c9b3c4-ab39-4a69-9da9-77c112548113
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 404 Not Found from http://localhost:8000/api/v1/exercises/77c9b3c4-ab39-4a69-9da9-77c112548113
INFO     BASE_ASSERTIONS:base.py:19 Check that response status code equals to 404
INFO     EXERCISES_ASSERTIONS:exercises.py:103 Check exercise not found response
INFO     ERRORS_ASSERTIONS:errors.py:63 Check internal error response
INFO     BASE_ASSERTIONS:base.py:38 Check that "details" equals to Exercise not found
INFO     SCHEMA_ASSERTIONS:schema.py:22 Validating JSON schema